Assignment two

lianml2025 opened the pull request:
<!-- PULL REQUEST TITLE: UofT-DSI | <Module Name> - Assignment <assignment number>-->
W3 SQL assignment 2
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Add on 2 PDF diagram and Edit SQL file
## What did you learn from the changes you have made?
SQL
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?

## How were these changes tested?
DB Browser
## A reference to a related issue in your repository (if applicable)

## Checklist
- [ Y] I can confirm that my changes are working as intended

lianml2025 committed:
Update Assignment2.md

finish section 1

lianml2025 committed:
Add files via upload

niyazmnazari reviewed (approved):
Thank you for submitting assignment 2. Section 1 looks correct but the queries has not been submitted.
Point: 50/70

lianml2025 commented:
Oh, I put the sql answers in the assignment2.sql file . Should the answers be extracted to another separate file?

For example, the screenshot shows the answer for the section 2 update sql answers.



Best,
minling Lian


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Niyaz Nazari ***@***.***>
Sent: Tuesday, August 19, 2025 8:00:55 AM
To: lianml2025/sql ***@***.***>
Cc: MINLING LIAN ***@***.***>; Author ***@***.***>
Subject: Re: [lianml2025/sql] Assignment two (PR #3)


@niyazmnazari approved this pull request.

Thank you for submitting assignment 2. Section 1 looks correct but the queries has not been submitted.
Point: 50/70

—
Reply to this email directly, view it on GitHub<https://github.com/lianml2025/sql/pull/3#pullrequestreview-3132007969>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BUYWBYFMZUIB7N2Q3LW4KAD3OMGXPAVCNFSM6AAAAACEA3SLB6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTCMZSGAYDOOJWHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

lianml2025 committed:
Update assignment2.sql

put the  answers to  assignment two folder, was put to main folder originally for submission

lianml2025 commented:
sorry I put my answers to main directory just copy to assignment two directory.

Sorry for any inconveniences.

Best,
Minling

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: MINLING LIAN ***@***.***>
Sent: Tuesday, August 19, 2025 8:11:30 AM
To: lianml2025/sql ***@***.***>; lianml2025/sql ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [lianml2025/sql] Assignment two (PR #3)

Oh, I put the sql answers in the assignment2.sql file . Should the answers be extracted to another separate file?

For example, the screenshot shows the answer for the section 2 update sql answers.



Best,
minling Lian


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Niyaz Nazari ***@***.***>
Sent: Tuesday, August 19, 2025 8:00:55 AM
To: lianml2025/sql ***@***.***>
Cc: MINLING LIAN ***@***.***>; Author ***@***.***>
Subject: Re: [lianml2025/sql] Assignment two (PR #3)


@niyazmnazari approved this pull request.

Thank you for submitting assignment 2. Section 1 looks correct but the queries has not been submitted.
Point: 50/70

—
Reply to this email directly, view it on GitHub<https://github.com/lianml2025/sql/pull/3#pullrequestreview-3132007969>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BUYWBYFMZUIB7N2Q3LW4KAD3OMGXPAVCNFSM6AAAAACEA3SLB6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTCMZSGAYDOOJWHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

lianml2025 commented:
Sorry just realized I put the answers in the main directory instead of assignment two directory. I just copied to the assignment two directory. Sorry for any inconvenience. Attached is the screenshot.


Best,
Minling

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Niyaz Nazari ***@***.***>
Sent: Tuesday, August 19, 2025 8:00:55 AM
To: lianml2025/sql ***@***.***>
Cc: MINLING LIAN ***@***.***>; Author ***@***.***>
Subject: Re: [lianml2025/sql] Assignment two (PR #3)


@niyazmnazari approved this pull request.

Thank you for submitting assignment 2. Section 1 looks correct but the queries has not been submitted.
Point: 50/70

—
Reply to this email directly, view it on GitHub<https://github.com/lianml2025/sql/pull/3#pullrequestreview-3132007969>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BUYWBYFMZUIB7N2Q3LW4KAD3OMGXPAVCNFSM6AAAAACEA3SLB6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTCMZSGAYDOOJWHE>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

niyazmnazari reviewed (approved):
Great job. I checked the queries as well.

70/70
